src/pkg/podman/podman.go

```diff
@@ -227,9 +227,16 @@ func IsToolboxImage(image string) (bool, error) {
 }
 
 // Pull pulls an image
-func Pull(imageName string) error {
+//
+// authfile is a path to a JSON authentication file and is internally used only
+// if it is not an empty string. The file at the path should exist.
+func Pull(imageName string, authfile string) error {
 	logLevelString := LogLevel.String()
-	args := []string{"--log-level", logLevelString, "pull", imageName}
+	args := []string{"--log-level", logLevelString, "pull"}
+    if authfile != "" {
+        args = append(args, []string{"--authfile", authfile}...)
+    }
+    args = append(args, imageName)
```

Comment:
The alignment is off.

*This was caught by `gofmt` in our test suite.  I have no idea why it eventually timed out.*